spec/models/review_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+
+  describe "model validations" do
+    it "requires a value" do
```

Comment:
I would replace "value" here with "rating" to show exactly which attribute of the review model you are validating.